### ISSUE TITLE
Logs: Download Robot Logs

### ISSUE THREAD

**Issue by umbhau** [Opentrons/opentrons#1350]
As a Run App user, I would like to be able to download my robot's logs from the Run App.

## Acceptance Criteria
- User able to download logs from robot settings page

## Implementation Notes
- Dependent on #1349 

## Design
- TBD

**Comment by umbhau** [comment]
Duplicate of #1727
